submissions/2021-03-29-OpenFF-Aniline-2D-Impropers-v1.0/README.md

```diff
@@ -0,0 +1,81 @@
+# OpenFF Aniline 2D Impropers v1.0:
+
+### Description
+
+This dataset contains a set of aniline derivatives which have para-substituted groups of varying electron donating and withdrawing properties. This dataset was curated in an effort to improve and understand improper torsions in force fields. We will scan the improper and proper angle simultaneously to better understand the coupling and energetics of these torsions.
+
+### General Information
+
+ - Date: 2021.03.29
+ - Class: OpenFF TorsionDrive
+ - Purpose: Substituted aniline derivatives with various electron withdrawing and donating groups
+ - Collection: TorsionDriveDataset
+ - Name: OpenFF Aniline 2D Impropers v1.0
+ - Number of unique molecules        24
+ - Number of filtered molecules      0
+ - Number of torsion drives          24
+ - Number of conformers min mean max 1   5.75 10
+ - Dataset Submitter: Simon Boothroyd
+ - Dataset Generator: Jessica Maat
+ - Set of charges: [-1.0, 0.0, 1.0]
+ - Mean molecular weight: 137.81
+ - Max molecular weight: 179.28
+ - Enumerate stereoisomers: False
+ - Enumerate tautomers: False
+ - Enumerate protomers: False
+
+
+### QCSubmit generation pipeline
+
+ - `generate-dataset`: The notebook used to generate the dataset from the input smiles set.
+
+### QCSubmit Manifest
+
+- `generate-smiles.py`: Molecule generation script.
```

Comment:
I don't see this file; is it committed? If the SMILES were hand-selected, won't be necessary.